cypress/integration/calculator-homework-Indre-Kelly/calculator.spec.js

```diff
@@ -0,0 +1,66 @@
+import Calculator from '../../page_objects/calculator_main/Calculator.js';
+
+var calculator = new Calculator;
+const builds = ["Prototype", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
+
+describe("Test Suit", () => {
```

Comment:
I know it was my suggestion to use _describe_ before tests, but for the future I suggest to use more meaningful test here e.g. 'Basic Calculator' (or something) similar instead of 'Test Suite'. In that way you will have more informative test reports :)